src/Form/Field.php

```diff
@@ -76,30 +78,6 @@ public function __construct(
 		$this->siblings = $siblings ?? new Fields([$this]);
 	}
 
-	/**
-	 * Returns field data
-	 */
-	public function data(bool $default = false): mixed
```

Comment:
`Question:` Should we first deprecate this (and inside the method call `::toStoredValue()`)?

Reply:
This is already happening. The method is now located in the Value mixin and marked as deprecated there.